versions.tf

```diff
@@ -2,6 +2,6 @@ terraform {
   required_version = ">= 0.13.0"
 
   required_providers {
-    aws = ">= 3.0"
+    aws = ">= 4.9.0"
```

Comment:
nit: It looks like `ephemeral_storage` was added in 4.8? [[1]](https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md#user-content-480-march-25-2022)